src/source/reference/get-user.rst

```diff
@@ -3,6 +3,38 @@
 
 GET /user
 =========
+:ref:`get_user_v3` | :ref:`get_user_v1`
```

Comment:
where's V2?

Reply:
I was wondering the same :D imho we don't have to publish it, but can add if preferred

Reply:
lol, we never had it in the docs?

Reply:
nope, only 2.5 years without doc on it :P

Reply:
ROFL ok merge it

Reply:
I need approval :P ➕